modules/OCPPConfiguration/main/util.hpp

```diff
@@ -0,0 +1,16 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Pionix GmbH and Contributors to EVerest
+
+#pragma once
+
+#include <c4/yml/std/map.hpp>
+#include <ryml.hpp>
+
+namespace module {
+class Util {
+public:
+    static ryml::Tree load_yaml_file(const std::string& file_path);
+    static void save_yaml_file(const std::string& file_path, ryml::Tree& tree);
+};
```

Comment:
classes shouldn't be used for namespacing,  just do
```
namespace module::util {
  ryml::Tree load_yaml_file(..);
  void save_yaml_file(..);
}
```
I probably also wouldn't expose all the ryml stuff.  Rather have a container or opaque data type for that.

Reply:
True. I changed that